selfdrive/athena/athenad.py

```diff
@@ -340,84 +381,89 @@ def do_reboot():
   return {"success": 1}
 
 
+UploadFilesToUrlResponse = Dict[str, Union[int, List[UploadItemDict], List[str]]]
+
+
 @dispatcher.add_method
-def uploadFileToUrl(fn, url, headers):
-  return uploadFilesToUrls([{
+def uploadFileToUrl(fn: str, url: str, headers: Dict[str, str]) -> UploadFilesToUrlResponse:
+  response: UploadFilesToUrlResponse = uploadFilesToUrls([{
```

Comment:
unfortunately mypy thinks this function is returning `Any` unless it is spelled out like this
```
selfdrive/athena/athenad.py:389: error: Returning Any from function declared to return "Dict[str, Union[int, List[Dict[str, Union[str, bool, int, float, Dict[str, str]]]], List[str]]]"
```

Reply:
I think this might be due to `jsonrpc.dispatcher` not having typing information, so Python doesn't know for sure that `uploadFilesToUrls` returns the same